application/jira/v1/config/tasks.json

```diff
@@ -0,0 +1,1181 @@
+          "integer"
+        ],
+        "instillUpstreamTypes": [
+          "value",
+          "reference",
+          "template"
+        ],
+        "type": "integer"
+      },
+      "update-history": {
+        "description": "Whether the action taken is added to the user's Recent history, as shown under `Your Work` in Jira.",
+        "title": "Update History",
+        "instillUIOrder": 5,
+        "instillFormat": "boolean",
+        "instillAcceptFormats": [
+          "boolean"
+        ],
+        "instillUpstreamTypes": [
+          "value"
+        ],
+        "type": "boolean"
+      }
+    },
+    "issue": {
+      "properties": {
+        "id": {
+          "description": "The ID of the issue",
+          "instillUIOrder": 0,
+          "title": "ID",
+          "instillFormat": "string",
+          "type": "string"
+        },
+        "key": {
+          "title": "Key",
+          "description": "The key of the issue, e.g. `JRA-1330`",
+          "instillShortDescription": "The key of the issue",
+          "instillUIOrder": 1,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        },
+        "self": {
+          "description": "The URL of the issue",
+          "instillUIOrder": 2,
+          "instillFormat": "string",
+          "title": "Self",
+          "type": "string"
+        },
+        "fields": {
+          "description": "The fields of the issue. All navigable and Agile fields are returned",
+          "instillUIOrder": 3,
+          "instillFormat": "semi-structured/json",
+          "title": "Fields",
+          "type": "object",
+          "required": []
+        },
+        "issue-type": {
+          "description": "The type of the issue, can be: `Task`, `Epic`",
+          "instillUIOrder": 4,
+          "instillFormat": "string",
+          "title": "Issue Type",
+          "type": "string"
+        },
+        "summary": {
+          "description": "The summary of the issue",
+          "instillUIOrder": 5,
+          "instillFormat": "string",
+          "title": "Summary",
+          "type": "string"
+        },
+        "description": {
+          "description": "The description of the issue",
+          "instillUIOrder": 6,
+          "instillFormat": "string",
+          "title": "Description",
+          "type": "string"
+        },
+        "status": {
+          "description": "The status of the issue, can be: `To Do`, `In Progress`, `Done`",
+          "instillUIOrder": 7,
+          "instillFormat": "string",
+          "title": "Status",
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "key",
+        "self",
+        "fields"
+      ],
+      "title": "Issue",
+      "type": "object"
+    },
+    "sprint": {
```

Comment:
question
In sprint, will we need to know which issues are in this sprint?

Reply:
Users can use `Issues of a sprint` in `list issues` task to achieve this task.